index.html

```diff
@@ -36,5 +38,218 @@ <h2>Software Developer</h2>
+                  </p>
+
+                  <div class="project-stack">
+                     <ul class="project-stack-list">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Buby</li>
+                     </ul>
+                  </div>
+
+                  <button class="project-btn">
+                     See Project
+                  </button>
+                  
+                  <!-- blur effect three -->
+                  <div class="blur-eff-three"></div>
+
+               </div>
+               <!-- end of project -->
+               <div class="project">
+                  <div class="project-display">
+
+                  </div>
+                  <h2 class="project-title">Multi-Post Stories</h2>
+                  <p class="project-description">
+                     A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                  </p>
+
+                  <div class="project-stack">
+                     <ul class="project-stack-list">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Buby</li>
+                     </ul>
+                  </div>
+
+                  <button class="project-btn">
+                     See Project
+                  </button>
+
+                  <!-- blur effect four -->
+                  <div class="blur-eff-four"></div>
+
+               </div>
+               <!-- end of project -->
+            </div>
+            <!-- end of projects container -->
+         </section>
+         <!-- end of projects section  -->
+
+         <!-- contact section -->
+         <section class="contact">
+            <div class="contact-text">
+               <h1 class="contact-title">
+                  contact us
+               </h1>
+               <p class="contact-description">
+                  If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+               </p>
+            </div>
+
+            <div class="contact-form">
+               <input type="text" placeholder="Text Input" class="user-text">
+               <input type="text" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email">
+               <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here"></textarea>
+               <button class="see-projects project-btn">
+                  See Project
+               </button>
```

Comment:
- Please work on the padding-left of the placeholders because of the text-input and email looks far pushed to the right compared to the Figma design 


##### Your design
![Screenshot from 2022-07-01 09-12-07](https://user-images.githubusercontent.com/58771507/176854162-3e9b88f1-a608-4eae-837f-03c1743555e7.png)

##### Figma design

![Screenshot from 2022-07-01 09-12-34](https://user-images.githubusercontent.com/58771507/176854192-ca29c580-9b7a-4679-b4d6-4e2efe833740.png)

Reply:
Sure, I will jump onto it. 
Thank You !!